cachito/workers/pkg_managers/gomod.py

```diff
@@ -33,6 +35,39 @@
 MODULE_VERSION_RE = re.compile(r"/v\d+$")
 
 
+def run_download_cmd(cmd: Iterable[str], params: Dict[str, str]) -> str:
+    """Run gomod command that downloads dependencies.
+
+    Such commands may fail due to network errors (go is bad at retrying), so the entire operation
+    will be retried a configurable number of times.
+
+    The backoff is constant. The download commands are typically used for multiple dependencies
+    at the same time, and the failure can be caused by any of them. Exponential backoff would make
+    more sense if we were downloading dependencies individually.
```

Comment:
Hmm, the question that Lui asked about reusing the same temp dir for the download (we do reuse one) has me wondering if exponential backoff + more retries wouldn't be worth it after all